### PR TITLE
Separate MatchProfileViewController from ProfileViewController

### PR DESCRIPTION
**Pull request by lucyxubroad**
## Overview

Before, we had a general profile VC. This PR breaks it up into one VC for showing your match for the week and another for general profile. Everything should still look the same.

### CODE REVIEW

File: Pear/Profile/MatchProfileViewController.swift [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        reachOutButton.snp.makeConstraints { make in
+            make.bottom.equalTo(view.safeAreaLayoutGuide).inset(reachOutPadding)
+            make.centerX.equalToSuperview()
+            make.size.equalTo(reachOutButtonSize)
+        }
+    }
+
+    @objc private func reachOutPressed() {
+        guard let pair = pair, let match = match, let user = user else { return }
+        let schedulingVC: SchedulingTimeViewController
```
Comment (danielVebman) [inline Pear/Profile/MatchProfileViewController.swift:166]: There's no reason to have this out here. Within each case just add `let` before `schedulingVC =`

File: Pear/Profile/MatchProfileViewController.swift [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+            reachOutButton.setTitle("Send email", for: .normal)
+        }
+
+        view.addSubview(reachOutButton)
+    }
+
+    private func setupConstraints() {
+        let reachOutPadding = LayoutHelper.shared.getCustomVerticalPadding(size: 70)
+        let meetupPadding = 24
+        let meetupWidth: CGFloat = 319
```
Comment (danielVebman) [inline Pear/Profile/MatchProfileViewController.swift:137]: Do you need the type?

File: Pear/Profile/MatchProfileViewController.swift [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                self.profileSections.append(.interests)
+            }
+            if pair.groups.count > 0 {
+                self.profileSections.append(.groups)
+            }
+            self.profileTableView.reloadData()
+        }
+    }
+
+    private func getPairThen(_ completion: @escaping (User) -> Void) {
```
Comment (danielVebman) [inline Pear/Profile/MatchProfileViewController.swift:71]: I'm not sure how idiomatic the `then` is but I don't feel strongly about it either way.

**Review by danielVebman — APPROVED** [review]
Noice 🚀

File: Pear/Profile/MatchProfileViewController.swift [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private var chatStatus: ChatStatus?
+    private let match: Match?
+    private var meetupStatusView: MeetupStatusView?
+    private var pair: User?
+    private var profileSections = [ProfileSectionType]()
+    private let profileTableView = UITableView(frame: .zero, style: .plain)
+    private let user: User?
+    private var reachOutButton = UIButton()
```
Comment (spd75) [inline Pear/Profile/MatchProfileViewController.swift:24]: Should this be alphabetized?  Everything else seems to be in alphabetical order besides reachOutButton

**Review by spd75 — APPROVED** [review]
Nice job!

**Pull request merged**